server/src/repositories/asset.repository.ts

```diff
@@ -1058,6 +1059,32 @@ export class AssetRepository {
       .executeTakeFirstOrThrow();
   }
 
+  @GenerateSql({
+    params: [{ libraryId: DummyValue.UUID, importPaths: [DummyValue.STRING], exclusionPatterns: [DummyValue.STRING] }],
+  })
+  async detectOfflineExternalSidecars(
```

Comment:
Sidecars are already treated as malleable, so I don't think sidecar handling should need any special cases for external.

Reply:
This is a speedup that we can do for external assets